speaker/bgp_controller.go

```diff
@@ -174,16 +174,22 @@ func (c *bgpController) ShouldAnnounce(l log.Logger, name string, _ []net.IP, po
 		return "notOwner"
 	}
 
-	if err := k8snodes.IsNodeAvailable(nodes[c.myNode]); err != nil {
```

Comment:
nit: I think it'd be cleaner (and more cherry-pick / backport friendly) to have a revert commit of the previous logic